client/src/App.tsx

```diff
@@ -21,6 +25,12 @@ function App() {
       <Adv />
       <MenuContainer />
       <CartContainer />
+      <Modal
+        isCancelBtn={true}
+        isOkBtn={true}
```

Comment:
물론 일단 테스트용으로 넣어두신 것 같긴합니다만!
```suggestion
        isCancelBtn
        isOkBtn
```

true인 props라면 이런식으로 shorthand usage가 있습니다!

Reply:
오 그렇네요 ㅋㅋㅋ 감사합니다 😀